blog/_posts/2017-11-10-linux-foundation-OSSummit.md

```diff
@@ -0,0 +1,41 @@
+---
+title: The Linux Foundation Open Source Summit 🇪🇺
+layout: post
+created_at: Wed Nov 11 2017
+permalink: blog/2017-11-15-KaradelaMarck-Linux-Foundation-OSSummit-Europe
+current: blog
+author: Kara de la Marck
+twitter: KaraMarck
+categories:
+- student-posts
+- conferences
+---
+Top resources discovered at the Open Source Summit that RGSoC alumni should consider applying to: [Outreachy](https://www.outreachy.org/) and the [Linux Foundation's LiFT scholarship programme](https://training.linuxfoundation.org/free-linux-training/linux-training-scholarship-program)! 🎉
+
+This year the [Linux Foundation’s Open Source Summit Europe](http://events.linuxfoundation.org/events/archive/2017/open-source-summit-europe) took place in beautiful Prague. Fortunately, I was able to attend thanks to RGSoC.
```

Comment:
A small nitpick in the last sentence: 'Fortunately, I was able to attend **it ,** thanks to RGSoC.'